src/Reads.cpp

```diff
@@ -37,6 +37,55 @@ void Reads::access(
 }
 
 
+void Reads::renameWithSuffix(const string& suffix) {
+    const string readsDataName = reads.getName();
+    const string readNamesDataName = readNames.getName();
+    const string readMetaDataDataName = readMetaData.getName();
+    const string readRepeatCountsDataName = readRepeatCounts.getName();
+    const string readFlagsDataName = readFlags.fileName;
+
+    reads.rename(readsDataName + suffix);
+    readNames.rename(readNamesDataName + suffix);
+    readMetaData.rename(readMetaDataDataName + suffix);
+    readRepeatCounts.rename(readRepeatCountsDataName + suffix);
+    readFlags.rename(readFlagsDataName + suffix);
+}
+
+
+void Reads::copyDataForReadsLongerThan(const Reads& rhs, uint64_t newMinReadLength) {
```

Comment:
This is probably very fast, but I would be curious to know how long it would typically take for a human assembly.

Reply:
I'm going to run a human genome assembly today. Planning to kill it once the reads are processed.

Reply:
See updated PR description.